### ISSUE TITLE
the repo clone is not done correctly

### ISSUE THREAD

**Issue by MateoLostanlen** [pyronear/pyro-sys-setup#27]
the repo clone is not done correctly. I haven't really tried to understand the problem, but it's probably a user issue.

I have this error:
`fatal: detected dubious ownership in repository at`
then by doing this 

I get [this](https://stackoverflow.com/questions/72978485/git-submodule-update-failed-with-fatal-detected-dubious-ownership-in-reposit) error 

then 

![image](https://github.com/pyronear/pyro-sys-setup/assets/17944639/8e44acf4-fb98-44b7-9196-2aa79dee2017)

then the git history is wrong

![image](https://github.com/pyronear/pyro-sys-setup/assets/17944639/2f31b71f-d13c-42de-9ec9-433640f9b70c)

in short I had to reclone the repo
